src/client/mds_client.cpp

```diff
@@ -114,6 +114,9 @@ LIBCURVE_ERROR MDSClient::MDSRPCExcutor::DoRPCTask(RPCFunc rpctask,
     // rpc超时时间
     uint64_t rpcTimeOutMS = metaServerOpt_.mdsRPCTimeoutMs;
 
+    // The count of normal retry
+    uint64_t nNormalRetry = 0;
```

Comment:
"n"NormalRetry， why doy  you use a letter n before this param?

Reply:
> "n"NormalRetry， why doy you use a letter n before this param?

The prefix `n` represent the count of normal retry.